web_src/js/components/RepoContributors.vue

```diff
@@ -0,0 +1,360 @@
+              <svg-icon name="octicon-triangle-down" :size="14"/>
+            </span>
+          </div>
+          <div class="menu">
+            <div :class="type === 'commits' ? 'active item' : 'item'">
+              {{ locale.contributionType.commits }}
+            </div>
+            <div :class="type === 'additions' ? 'active item' : 'item'">
+              {{ locale.contributionType.additions }}
+            </div>
+            <div :class="type === 'deletions' ? 'active item' : 'item'">
+              {{ locale.contributionType.deletions }}
+            </div>
+          </div>
+        </div>
+      </div>
+    </h2>
+    <div class="divider"/>
+    <div class="gt-df main-graph">
+      <div v-if="isLoading || errorText !== ''" class="gt-tc gt-m-auto">
+        <div v-if="isLoading">
+          <SvgIcon name="octicon-sync" class="gt-mr-3 job-status-rotate"/>
+          {{ locale.loadingInfo }}
+        </div>
+        <div v-else class="text red">
+          <SvgIcon name="octicon-x-circle-fill"/>
+          {{ errorText }}
+        </div>
+      </div>
+      <CLine
+        v-memo="[totalStats.weeks, type]" v-if="Object.keys(totalStats).length !== 0"
+        :data="toGraphData(totalStats.weeks)" :options="getOptions('main')"
+      />
+    </div>
+    <div class="divider"/>
+
+    <div class="ui attached two column grid">
+      <div
+        v-for="(contributor, index) in sortedContributors" :key="index" class="column stats-table"
+        v-memo="[sortedContributors, type]"
+      >
+        <div class="ui top attached header gt-df gt-f1">
+          <b class="ui right">#{{ index + 1 }}</b>
+          <a :href="contributor.home_link">
+            <img height="40" width="40" :href="contributor.avatar_link" :src="contributor.avatar_link">
+          </a>
+          <div class="gt-ml-3">
+            <a :href="contributor.home_link"><h4>{{ contributor.name }}</h4></a>
+            <p class="gt-font-12">
+              <strong>{{ contributor.total_commits.toLocaleString() }} {{ locale.contributionType.commits }}</strong>
+              <strong class="text green">{{ contributor.total_additions.toLocaleString() }}++ </strong>
+              <strong class="text red">
+                {{ contributor.total_deletions.toLocaleString() }}--</strong>
+            </p>
+          </div>
+        </div>
+        <div class="ui attached segment">
+          <div>
+            <CLine
+              :data="toGraphData(contributor.weeks)"
+              :options="getOptions('contributor')"
+            />
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import {SvgIcon} from '../svg.js';
+import {
+  Chart,
+  Title,
+  Tooltip,
+  Legend,
+  BarElement,
+  CategoryScale,
+  LinearScale,
+  TimeScale,
+  PointElement,
+  LineElement,
+  Filler,
+} from 'chart.js';
+import zoomPlugin from 'chartjs-plugin-zoom';
+import {Line as CLine} from 'vue-chartjs';
```

Comment:
Oh, I renamed the name. I agree rename it to `ChartLine`.

Reply:
There was some problems occuring if I don't rename IIRC. But you are right, it would be probably better if I rename it to `ChartLine`. I'll do it in next commit.